src/components/buttons/IconButton/types.ts

```diff
@@ -4,28 +4,38 @@
 import { ButtonProps } from '../Button/types'
 import AddIcon from '@mui/icons-material/Add'
 import CancelIcon from '@mui/icons-material/Cancel'
+import CloseIcon from '@mui/icons-material/Close'
 import DeleteIcon from '@mui/icons-material/Delete'
 import CloudDownloadIcon from '@mui/icons-material/CloudDownload'
 import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward'
 import EditIcon from '@mui/icons-material/Edit'
 import VisibilityIcon from '@mui/icons-material/Visibility'
 import SaveIcon from '@mui/icons-material/Save'
 import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import FilterListIcon from '@mui/icons-material/FilterList'
+import PostAddIcon from '@mui/icons-material/PostAdd'
 
 export type FontSize = 'inherit' | 'small' | 'medium' | 'large'
 
-export type ButtonType = 'add' | 'cancel' | 'delete' | 'download' | 'downward' | 'edit' | 'view' | 'save' | 'upward'
+export type ButtonType = 'add' | 'cancel' | 'close' | 'delete' | 'download' | 'downward' | 'edit' | 'view' | 'save' | 'upward' | 'expandLess' | 'expandMore' | 'filterList' | 'postAdd'
```

Comment:
1. `filterList` si `postAdd` sunt prea specifice pentru a fi adaugate in lista de `type` din IconButton.
2. Scoate `close` si inlocuieste icon-ul de la `cancel` cu ce ai pus la `close`.
3. Exista un story ptr IconButton, Types, in care trebuie adaugate cele doua type-uri noi, `expandMore`, `expandLess` https://osstotalsoft.github.io/rocket-ui-ts/?path=/docs/components-buttons-iconbutton--docs#types